MAUI/TreeMap/migration.md

```diff
@@ -989,11 +989,13 @@ public class PopulationViewModel
 
 </table>
 
-### TreeMapRangeBrushSettings
+### RangeBrushSettings
 
-To categorize leaf nodes in a TreeMap according to different ranges of background color values, use `TreeMapRangeBrushSettings`. Define the upper and lower limits of each range using the `To` and `From` properties of `TreeMapRangeBrush`.
+To categorize leaf items in a TreeMap according to different ranges of background color values, use `TreeMapRangeBrushSettings`. Define the upper and lower limits of each range using the `To` and `From` properties of `TreeMapRangeBrush`.
 
-#### TreeMapRangeBrush
+#### RangeBrush
+
+The following code example, explains how to configure the range brush in .NET MAUI SfTreeMap control.
```

Comment:
The following code example explains how to configure the range brush in .NET MAUI SfTreeMap control.

Reply:
Changed content